### PR TITLE
Applied MUI on the GPT list component

### PR DESCRIPTION
**Pull request by Tianyu82**
Applied MUI on the GPT list component

### CODE REVIEW

**Comment by MooyGhul** [comment]
There are merge conflicts

**Comment by Tianyu82** [comment]
I solved the conflicts in the package-lock.json file

File: client/src/pages/gptsListPage.tsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
 import ListOfUsersGPT from '../components/lists';
 import CreateGPTBtn from '../components/createNewGPTBtn';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
```
Comment (MooyGhul) [inline client/src/pages/gptsListPage.tsx:6]: I think it can be something like
import {Box, Button, ... } from '@mui/material';

**Review by MooyGhul — APPROVED** [review]
LGTM with nit

**Comment by Tianyu82** [comment]
The MUI is done

**Pull request merged**